Jenkinsfile

```diff
@@ -1,28 +1,60 @@
 pipeline {
    agent any
 
-//    tools {
-      // Install the Maven version configured as "M3" and add it to the path.
-//       maven "maven 3"
-//    }
-
    stages {
       stage('Build') {
+
+
          steps {
             // Get some code from a GitHub repository
-            git 'https://github.com/archick12/python_automation.git'
+            git branch:'jenkins_integration', url:'https://github.com/vitaliisotnichenko/python_automation'
```

Comment:
Please, replace hard code with dynamic branch name https://stackoverflow.com/questions/32789619/jenkins-multibranch-pipeline-what-is-the-branch-name-variable